draft-ietf-wimse-arch.md

```diff
@@ -75,6 +75,10 @@ Identity proxy is an intermediary that can inspect, replace or augment workload
 
 Attestation is the function through which a task verifies the identity of a separate Workload. (TBD: sync definition with reference RaTS architecture)
 
+* Workload Identity Token
+
+An authentication token that contains a workload identifier used for service to service authentication. The token is bound to a cryptographic key and requires that the presenter provide proof of possession of the secret key material. This token is further defined in {{?I-D.ietf-wimse-s2s-protocol}}
```

Comment:
Let's drop first "authentication". That is,

A token that contains a workload identifier used for service to service authentication. It is bound to a cryptographic key and requires that the presenter provide proof of possession of the secret key material. This token is further defined in {{?I-D.ietf-wimse-s2s-protocol}}

Reply:
Updated